ruby/vampires.rb

```diff
@@ -0,0 +1,136 @@
+# Gathering vampire data with puts statements, and assigning user input
+# to variables
+
+def new_employee
+	puts "What is your name?"
+		name = gets.chomp
+	puts "How old are you?"
+		age = gets.chomp
+		#convert answer to integer
+		age = age.to_i
+	puts "What year were you born?"
+		birth_year = gets.chomp
+		#convert answer to integer
+		birth_year = birth_year.to_i
+	#compare age to birth_year and assign boolean value 
+		if (2016 - birth_year) == age
```

Comment:
I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.